emulators/mame/Portfile

```diff
@@ -23,6 +34,68 @@ long_description    The purpose of MAME is to preserve decades of software histo
     the hardware and how it functions. The source code to MAME serves as this \
     documentation.
 
+github.tarball_from archive
+
+#------------------------------------------------------------------------------
+# Patches and Checksums
+#------------------------------------------------------------------------------
+
+# Patch 'language-portuguese_brazil' is only applicable to release 0.226.
+# Fix was submitted, and merged, by Mame maintainers... and will be part of
+# official Mame releases from 0.227 onward. Net-Net, this particular patch can
+# be removed starting with 0.227. Tracked by Mame issue:
+# https://github.com/mamedev/mame/issues/7510
+#
+# Conversely, patch 'scripts-genie-compile-warnings' is not specific to any
+# particular Mame release. Indeed, it is technically not required at all.
+# However, it eliminates spurious warnings due to use of deprecated types
+# and/or calls... and reduces pollution in the build log.
+
+patchfiles          mame-patch-0226-language-portuguese_brazil.diff \
+                    mame-patch-0226-scripts-genie-compile-warnings.diff
+
+checksums           mame-0226.tar.gz \
```

Comment:
There is no need to name the download file. Please remove that (`mame-0226.tar.gz`). Otherwise this has to be updated everyone version update and that is not desirable.

Reply:
@macsguy - you only need to name your checksums if you have more than one.

Reply:
Ah, this was a leftover from a few commits back, when I removed the checksums for all of the other artifacts.  :-)

Reply:
Change committed, along with switching Python default from 3.9 to 3.8.